mopidy/mpd/actor.py

```diff
@@ -35,17 +37,17 @@ def __init__(self, config, core):
         super(MpdFrontend, self).__init__()
 
         self.hostname = network.format_hostname(config['mpd']['hostname'])
-        self.port = config['mpd']['port']
+        self.port = config['mpd']['port'] if 'port' in config['mpd'] else None
```

Comment:
Anything defined in the config's schema should always be defined in the dictionary (and set to None if not otherwise assigned a value by any other means). So I don't think you need this.